### PR TITLE
Create applications

### PR DESCRIPTION
**Pull request by buehner**
This PR contains a first backend implementation for the creation of new webapps and introduces or changes the following:
- Use an `ApplicationData` data transfer object (for easy deserialization of application data (JSON) that is sent from the client to the backend)
- Restructuring of the init beans xml (with new template/prototype beans)
- Web interface for the app creation
- Service implementation

Currently, the implementation only supports the following things to be sent by the client:
- General information (name, description, language etc)
- Startview

What's still missing is the ability to configure tools and layers (!) for a new webapp

### CODE REVIEW

File: src/main/java/de/terrestris/momo/service/MomoApplicationService.java [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+	private static final Set<String> DEFAULT_MAP_CONTROLS = Collections.unmodifiableSet(
+			new HashSet<String>(Arrays.asList("Attribution", "Zoom", "Rotate", "ScaleLine", "ZoomSlider")));
+
+	private static final List<String> DEFAULT_MAP_CONTAINER_MODULES = Collections.unmodifiableList(Arrays
+			.asList("Show meta toolbar button", "Meta info toolbar for the map", "Tools Toolbar", "Gazetteer Grid"));
+
+	private static final List<String> DEFAULT_VIEWPORT_MODULES = Collections
+			.unmodifiableList(Arrays.asList("Viewport Header", "Legend Tree", "Gazetteer Grid"));
+
```
Comment (annarieger) [inline src/main/java/de/terrestris/momo/service/MomoApplicationService.java:83]: Hmmm, it's probably not the best approach to use here the hardcoded module names as they can be modified/deleted/renamed.. 

But as we don't have anything else at the moment you can leave it firstly as is.. Just keep it in mind, that could be a problem lately..

Comment (buehner) [reply src/main/java/de/terrestris/momo/service/MomoApplicationService.java:83]: Yes. I know, but there was no time to make it better in that short time :-(

Comment (annarieger) [reply src/main/java/de/terrestris/momo/service/MomoApplicationService.java:83]: Sadly, but you're right..

**Comment by annarieger** [comment]
Thx @buehner for your work. Could you please merge this PR? I'm +1

**Pull request merged**